website/sidebars.js

```diff
@@ -468,7 +469,7 @@ const sidebarSettings = {
         "docs/collaborate/collaborate-with-others",
         {
           type: "category",
-          label: "Explore dbt projects",
+          label: "Navigate dbt Explorer",
```

Comment:
suggested by katherine
```suggestion
          label: "Discover with dbt Explorer",
```